src/main/java/de/tum/in/www1/artemis/web/rest/FileResource.java

```diff
@@ -167,6 +164,106 @@ public ResponseEntity<byte[]> getTemplateFile(@PathVariable ProgrammingLanguage
         return getTemplateFileContentWithResponse(languagePrefix, projectTypePrefix);
     }
 
+    /**
+     * GET /files/aeolus/templates/:language/:projectType : Get the aeolus template file with the given filename<br/>
+     * GET /files/aeolus/templates/:language : Get the aeolus template file with the given filename
+     * <p>
+     * The windfile contains the default build plan configuration for new programming exercises.
+     *
+     * @param language       The programming language for which the aeolus template file should be returned
+     * @param projectType    The project type for which the template file should be returned. If omitted, a default depending on the language will be used.
+     * @param staticAnalysis Whether the static analysis template should be used
+     * @param sequentialRuns Whether the sequential runs template should be used
+     * @param testCoverage   Whether the test coverage template should be used
+     * @return The requested file, or 404 if the file doesn't exist
+     */
+    @GetMapping({ "files/aeolus/templates/{language}/{projectType}", "files/aeolus/templates/{language}" })
+    @EnforceAtLeastEditor
+    public ResponseEntity<String> getAeolusTemplate(@PathVariable ProgrammingLanguage language, @PathVariable Optional<ProjectType> projectType,
+            @RequestParam(value = "staticAnalysis", required = false) String staticAnalysis, @RequestParam(value = "sequentialRuns", required = false) String sequentialRuns,
+            @RequestParam(value = "testCoverage", required = false) String testCoverage) {
```

Comment:
```suggestion
            @RequestParam(value = "staticAnalysis", required = false) boolean staticAnalysis, @RequestParam(value = "sequentialRuns", required = false) boolean sequentialRuns,
            @RequestParam(value = "testCoverage", required = false) boolean testCoverage) {
```
Can’t Spring do automatic parsing/conversion for boolean request parameters?

Reply:
thanks! I changed it to that